src/index.js

```diff
@@ -0,0 +1,26 @@
+var products = document.querySelector(".product-list");
```

Comment:
Tudo que for elemento é interessante colocar `nomeDoNoElement` -> `productsElement`.

Reply:
ok